src/main/java/com/loadero/model/File.java

```diff
@@ -0,0 +1,101 @@
+package com.loadero.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.loadero.Loadero;
+import com.loadero.http.ApiResource;
+import com.loadero.http.RequestMethod;
+import java.io.IOException;
+
+/**
+ * Used to read information about a project file.
+ */
+public final class File {
```

Comment:
i'm thinking that with media file removal we actually already have a breaking change here, so might as well solve this one properly by replacing the script type. is there anything else we might be deprecating in the near future?